bgpd/bgp_flowspec_vty.c

```diff
@@ -300,7 +300,9 @@ void route_vty_out_flowspec(struct vty *vty, const struct prefix *p,
 	if (!path)
 		return;
 
-	ipv6_ecomm = bgp_attr_get_ipv6_ecommunity(path->attr);
+	if (path->attr)
+		ipv6_ecomm = bgp_attr_get_ipv6_ecommunity(path->attr);
+
 	if (path->attr && (path->attr->ecommunity || ipv6_ecomm)) {
```

Comment:
ipv6_ecomm isn't initialized (to NULL) ?

Reply:
fixed :)